### PR TITLE
Fix #2289: Management provider information lost after failover

### LINKED ISSUE #2289: Issue with PoolSettingsProvider and PoolStatisticsProvider after failover
After a fail over, there is no longer pool server side stats nor capabilities.

### PR DESCRIPTION
**Pull request by mathieucarbou**
Requires https://github.com/Terracotta-OSS/terracotta-platform/pull/445

See https://github.com/Terracotta-OSS/terracotta-platform/pull/445 for explanations

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -7,7 +7,7 @@ slf4jVersion = 1.7.25
 sizeofVersion = 0.3.0
 
 # Terracotta clustered
-terracottaPlatformVersion = 5.4.0-pre26
+terracottaPlatformVersion = 5.4-SNAPSHOT
```
Comment (mathieucarbou) [inline gradle.properties]: will update the version + mvnlocal when https://github.com/Terracotta-OSS/terracotta-platform/pull/445 will be merged and released

File: clustered/integration-test/src/test/resources/logback-ext.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Copyright (c) 2011-2018 Software AG, Darmstadt, Germany and/or Software AG USA Inc., Reston, VA, USA, and/or its subsidiaries and/or its affiliates and/or their licensors.
+  Use, reproduction, transfer, publication or disclosure is prohibited except as specifically provided for in your License Agreement with Software AG.
+  -->
+
+<included>
+
```
Comment (myronkscott) [inline clustered/integration-test/src/test/resources/logback-ext.xml]: you sure you want all this logging?

Comment (mathieucarbou) [reply clustered/integration-test/src/test/resources/logback-ext.xml]: no I will remove

Comment (mathieucarbou) [reply clustered/integration-test/src/test/resources/logback-ext.xml]: no ;-) I need to remove this.

**Comment by anthonydahanne** [comment]
@mathieucarbou : does not compile ?
```/scratch/jenkins/workspace/ehcache3-pull-requests/jdk/Oracle JDK 1.8(latest)/label_exp/standard/testVM/8/clustered/server/src/main/java/org/ehcache/clustered/server/management/Management.java:88: error: cannot find symbol
09:17:00       managementRegistry.entityCreated();```

**Comment by mathieucarbou** [comment]
@anthonydahanne : see PR description... It needs https://github.com/Terracotta-OSS/terracotta-platform/pull/445 and a release first!

**Comment by mathieucarbou** [comment]
test this please

**Pull request merged**